src/site/content/en/blog/fugu-status/index.md

```diff
@@ -27,7 +27,7 @@ the [Fugu API Tracker](https://goo.gle/fugu-api-tracker).
 These APIs are only available behind a flag. They're experimental and still
 under development. They are not ready for use in production. There's a good
 chance there are bugs, that these APIs will break, or the API surface will
-change.
+change.<a name="shape-face-text"></a>
```

Comment:
What's up with this content-less anchor? Also, it seems out of place?

Reply:
This is how you create a place to link to in HTML. There are two of these connected to the entries that are related so that readers can click between them rather than having to scroll and search.

The placing of these is a hack. If I put the anchors where they would seem logical code-wise, the resulting viewport placement completely or partially obscures the content I'm intending to link to. I think this might be solvable with CSS, but I don't know how to do it. I meant to open a ticket along with this PR.